src/components/PercentContainer/PercentContainer.jsx

```diff
@@ -37,3 +37,5 @@ PercentContainer.propTypes = {
   games: PropTypes.number,
   percent: PropTypes.number,
 };
+
+export default PercentContainer;
```

Comment:
there was no default export in this file, but it was expected; new eslint rules find this